low-level/files/opt/seagate/sspl/setup/sspl_post_install.py

```diff
@@ -128,14 +130,17 @@ def process(self):
         if self.PRODUCT_NAME == "LDR_R1":
             # setup consul if not running already
             if not os.path.exists(REPLACEMENT_NODE_ENV_VAR_FILE):
-                sspl_setup_consul = "%s/sspl_setup_consul -e %s" % (self._script_dir,
-                                                                    self.ENVIRONMENT)
-                output, error, returncode = SimpleProcess(sspl_setup_consul).run()
+                sspl_setup_consul = "%s/sspl_setup_consul -e %s" % (
+                    self._script_dir,
+                    self.ENVIRONMENT)
+                output, error, returncode = SimpleProcess(
+                    sspl_setup_consul).run()
                 if returncode != 0:
                     raise SetupError(returncode, error, sspl_setup_consul)
 
         # Install packages which are not available in YUM repo, from PIP
-        pip_cmd = "python3 -m pip install -r %s/low-level/requirements.txt" % (SSPL_BASE_DIR)
+        pip_cmd = "python3 -m pip install -r %s/low-level/requirements.txt" % (
```

Comment:
this approach is going away, no components would be doing its dependency installation

Reply:
post-install will only have validations only. Rebase with main branch, that would update code

Reply:
Done